frontend/src/app/sessions/active-sessions/file-browser/file-browser.component.html

```diff
@@ -111,7 +111,7 @@ <h1>File browser (experimental)</h1>
   </mat-nested-tree-node>
 </mat-tree>
 
-<mat-dialog-actions>
+<div>
```

Comment:
The file browser component definitely misses some padding right now. So I think wrapping everything in a `div` with the `dialog` class would be sufficient here.